### PR TITLE
PT-847 Change logo in pdf for infusion site

### PR DESCRIPTION
**Pull request by franck-fourel**

### CODE REVIEW

File: src/modules/print/BasicsPrintView.js [hunk 1]
```diff
@@ -698,6 +706,8 @@ class BasicsPrintView extends PrintView {
         return {
           color: this.colors[type],
           count: _.get(data, `${day.date}.total`, _.get(data, `${day.date}.count`, 0)),
+          // translation missing
```
Comment (yvon-dblg) [inline src/modules/print/BasicsPrintView.js]:
```suggestion
          // FIXME translation missing
```
Just to identify this comment purpose.

**Review by yvon-dblg — COMMENTED** [review]
A comment on a comment !
For the rest: LGTM

**Comment by franck-fourel** [comment]
> A comment on a comment !
> For the rest: LGTM

thks
Can you please approve it

**Review by yvon-dblg — APPROVED** [review]
LGTM

**Pull request merged**